src/platform/backends/qemu/qemu_virtual_machine.cpp

```diff
@@ -224,11 +289,29 @@ void mp::QemuVirtualMachine::shutdown()
     }
     else
     {
-        vm_process->terminate();
+        vm_process->kill();
```

Comment:
Why this change?

Reply:
I have found in some cases that the qemu process is in some state where it ignore the `SIGTERM`, so we have to wait 30s before Qt will actually just kill the process.  Since we really don't care about the process at this point, a `kill()` just gets the job done and we don't have to wait.